### DISCUSSION TITLE
Deploy on Github Pages

### DISCUSSION THREAD

**Discussion by Ayuyon** [community/community#69026] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Does someone have any tutorial that can help to deploy an app on Github pages?

**Comment by xtycoonfi** [comment]
Hi! 👋

Sure, here : 

- [Deploy Your Portfolio Site to GitHub Pages for Free!](https://www.youtube.com/watch?v=XGcuxuhV-Jg&ab_channel=Covalence)
- [How to Host a Website On Github Pages](https://www.youtube.com/watch?v=OltY8JIaP-4&ab_channel=KennyYipCoding)
- [How To Deploy A React App To Github Pages (Simple)](https://www.youtube.com/watch?v=Q9n2mLqXFpU&ab_channel=PedroTech)

Hope it will help you!

**Reply by Ayuyon** [reply]
I will follow the tutorial thank you very much!

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
